.ci/pipeline.yml

```diff
@@ -0,0 +1,487 @@
+shared:
+  - &slack-alert
+    author_name: cycloid.io
+    author_link: https://www.cycloid.io/
+    title_link: https://console.cycloid.io/projects/((customer))/((project))/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_ID
+    fields:
+      - title: Project
+        value: ((project))
+        short: true
+      - title: Environment
+        value: ((env))
+        short: true
+
+  - &run-unit-tests
+    args:
+    - '-ec'
+    - |
+      export GOPATH=${PWD}/go
+      cd code
+      ((script))
+
+groups:
+  - name: all
+    jobs:
+      - test-pr
+      - test-((branch))
+      - version-((branch))
+      - release-((branch))
+      - publish-docker-image-master
+      - bump-to-next-patch-version
+      - bump-to-next-minor-version
+      - bump-to-next-major-version
+
+  - name: tests
+    jobs:
+      - test-pr
+      - test-((branch))
+
+  - name: release
+    jobs:
+      - test-((branch))
+      - version-((branch))
+      - release-((branch))
+      - publish-docker-image-master
+      - bump-to-next-patch-version
+      - bump-to-next-minor-version
+      - bump-to-next-major-version
+
+resource_types:
+  - name: slack-notification
+    type: docker-image
+    source:
+      repository: cfcommunity/slack-notification-resource
+      tag: latest
+
+  - name: pull-request
+    type: docker-image
+    source:
+      repository: teliaoss/github-pr-resource
+      tag: v0.15.0
+
+  - name: report-github-status
+    type: docker-image
+    source:
+      repository: cycloid/github-status-resource
+      tag: latest
+
+  - name: cycloid-events
+    type: docker-image
+    source:
+      repository: cycloid/cycloid-events-resource
+      tag: latest
+    privileged: false
+
+resources:
+  - name: report-status-((branch))
+    type: report-github-status
+    source:
+      access_token: ((github-access-token))
+      branch: ((branch))
+      repository: ((github-repo-owner))/((github-repo-name))
+
+  - name: golang-image
+    type: docker-image
+    source:
+      repository: golang
+      tag: 1.12
+
+  - name: cycloid-events
+    type: cycloid-events
+    source:
+      api_login: ((cycloid_bot_account.login))
```

Comment:
The name of the vault cred would be better into variables.yml

So in this file use something like `((event_login))`
And put into variables.yml 

```
event_login: ((cycloid_bot_account.login))
```